src/attr.rs

```diff
@@ -304,10 +304,10 @@ impl<'s> IntoIterator for &'s Attributes<'s> {
 }
 
 /// Iterator over mutable references to [Attributes] key-value pairs, in arbitrary order.
-pub struct AttributesIterMut<'s>(std::slice::IterMut<'s, (&'s str, AttributeValue<'s>)>);
+pub struct AttributesIterMut<'a, 's>(std::slice::IterMut<'a, (&'s str, AttributeValue<'s>)>);
 
-impl<'s> Iterator for AttributesIterMut<'s> {
-    type Item = (&'s mut &'s str, &'s mut AttributeValue<'s>);
+impl<'s, 'a: 's> Iterator for AttributesIterMut<'a, 's> {
```

Comment:
Similarly here, we do not want the iterator (`'a`) to outlive the source code (`'s`) here. This can be changed to `<'s, 'a>`